cpu/ppc64_cpu_test.py

```diff
@@ -41,11 +41,13 @@ def setUp(self):
         """
         if 'ppc' not in distro.detect().arch:
             self.cancel("Processor is not ppc64")
-        if SoftwareManager().check_installed("powerpc-utils") is False:
-            if SoftwareManager().install("powerpc-utils") is False:
+        self.sm = SoftwareManager()
+        if self.sm.check_installed("powerpc-utils") is False:
+            if self.sm.install("powerpc-utils") is False:
```

Comment:
if not self.sm.check_installed("powerpc-utils"): will work ?

Reply:
should work. I did not touch the code(except for variable scope change) and reused the original code. Will include the change in next version